apps/site/src/app/apply/sections/Form/SchoolInformation.tsx

```diff
@@ -0,0 +1,80 @@
+import styles from "./Form.module.scss";
+import DropdownSelect from "../Components/dropdownSelect";
+
+//I'm not sure if we want to have different submission values
+//than display texts for the inputs, so I seperated both
+//for good measure
```

Comment:
This is fine! We'll need to process these in the backend, so we should keep them separate as you have it.